Source/Core/Core/AchievementManager.h

```diff
@@ -76,6 +76,10 @@ class AchievementManager
   using RichPresence = std::array<char, RP_SIZE>;
   using Badge = VideoCommon::CustomTextureData::ArraySlice::Level;
   static constexpr size_t MAX_DISPLAYED_LBOARDS = 4;
+  // This is hardcoded to 24MB for now for RetroAchievements testing and to synchronize with current
+  // rcheevos and may change or become a variable in the future.
```

Comment:
I feel like this comment is burying the important part a bit. What's important for a person who sees this to know is:

* Why is this hardcoded?
* Is it okay to make this not hardcoded? Does something else need to be fixed first before we can make this not hardcoded, and if so, what?

Therefore I think it would be better to start off with a sentence like "This is hardcoded to 24 MiB because rcheevos hardcodes it to 24 MiB." Information along the lines of "this is being done for now for testing" or "may change in the future" often lingers around and gets outdated, but you can include that too afterwards if you want to.